cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"gopkg.in/urfave/cli.v1"
+)
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "livepeer-cli"
+	app.Usage = "interact with local Livepeer node"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "http",
+			Usage: "local http port",
+			Value: "8935",
+		},
+		cli.StringFlag{
+			Name:  "rtmp",
+			Usage: "local rtmp port",
+			Value: "1935",
+		},
+		cli.IntFlag{
+			Name:  "loglevel",
+			Value: 4,
+			Usage: "log level to emit to the screen",
+		},
+	}
+	app.Action = func(c *cli.Context) error {
+		// Set up the logger to print everything and the random generator
+		log.Root().SetHandler(log.LvlFilterHandler(log.Lvl(c.Int("loglevel")), log.StreamHandler(os.Stdout, log.TerminalFormat(true))))
+		rand.Seed(time.Now().UnixNano())
+
+		// Start the wizard and relinquish control
+		w := &wizard{
+			endpoint: fmt.Sprintf("http://localhost:%v/status", c.String("http")),
```

Comment:
Could probably add a `host` flag and replace `localhost` with the host passed in so this would work with remote nodes as well.

Reply:
Yeah - I was thinking maybe we can use a config file that saves all of the configs.